R/get_predicted.R

```diff
@@ -945,7 +946,7 @@ get_predicted.phylolm <- function(x,
       stringsAsFactors = FALSE
     )
     # make sure we have the correct name for the "Response column"
-    if (is_wiener) {
+    if (is_wiener || is_lnr) {
```

Comment:
I think we could have something more generic than is_lnr, like `is_ssm` ("sequential sampling models", the official name of decision making models)

Reply:
or `is_rtchoice`

Reply:
We could also change `is_wiener` to the general name